linera-core/src/worker.rs

```diff
@@ -197,8 +197,12 @@ pub enum WorkerError {
     },
     #[error("Cannot confirm a block before its predecessors: {current_block_height:?}")]
     MissingEarlierBlocks { current_block_height: BlockHeight },
-    #[error("{epoch:?} is not recognized by chain {chain_id:}")]
-    InvalidEpoch { chain_id: ChainId, epoch: Epoch },
+    #[error("Unexpected epoch {epoch:}; chain {chain_id:} is at {chain_epoch:}")]
```

Comment:
Did you mean `:` instead of `;` ?

Reply:
Not really; but maybe `:` is more appropriate. :+1: